pkg/snapshotter/snapshotter_csi.go

```diff
@@ -283,6 +286,14 @@ func (c *csiDriver) SnapshotStatus(name, namespace string) (SnapshotInfo, error)
 	if contentName != "" {
 		vscError = c.getSnapshotContentError(contentName)
 	}
+
+	snapshotTimeout, err := getSnapshotTimeout()
```

Comment:
If this fails, should we log a msg  and fall back to the default one and not fail the backup?

Reply:
Is there somewhere we can indicate this to the customer? I agree breaking all snapshots backups is a drastic reaction to a bad value. If the customer is having issues still with the default value they should still notice, and the damage would be contained

Reply:
@lpitstickpstg Other than logging to stork POD logs we don't have any other way to report it back. And the default value is 5 mins hard-coded , unless someone tampers that strings we have no way of failing in here.  If developer tampers that value it eventually fails in switch case.
Hence I feel we can adopt the default value in case the user provided timeout  value is a malformed one.
@prashanthpx @lpitstickpstg Kindly let me know your opinion